src/site/cookies/index.njk

```diff
@@ -0,0 +1,101 @@
+---
+title: Cookies on the MOD.UK Design System
+description: How we use cookies to make the MOD.UK Design System work and collect information about how you use this website.
+tags: footer
+order: 300
+---
+
+{%- extends "layouts/_base.njk" -%}
+
+{% from "moduk/components/table/macro.njk" import modukTable %}
+{% from "moduk/components/notification-banner/macro.njk" import modukNotificationBanner %}
+
+{% set html %}
```

Comment:
Would be good to make the name of this variable more specific e.g. `notificationBannerHtml`:

```suggestion
{% set notificationBannerHtml %}
```